plugins/controlled-documents-resources/package.json

```diff
@@ -34,7 +34,8 @@
     "ts-jest": "^29.1.1",
     "@types/jest": "^29.5.5",
     "eslint-plugin-svelte": "^2.35.1",
-    "svelte-eslint-parser": "^0.33.1"
+    "svelte-eslint-parser": "^0.33.1",
+    "yjs": "^13.5.52"
```

Comment:
why we need it?

Reply:
We import type for YDoc for documents diff view. It was in dependencies and I moved it to devDependencies as only used for types.

Reply:
Do we really need it?

Reply:
Hid it in text-editor and removed yjd dependency from controlled documents.